### ISSUE TITLE
dilatation of pulmonary veins

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#7700]
**Preferred term label:**  Pulmonary vein varix (PVV)
**Synonyms** Pulmonary vein dilatation

**Definition (free text, please give PubMed ID)**
Pulmonary vein varix (PVV), refers to a localised dilatation of a pulmonary vein. PVV is a rare condition and can be congenital or acquired. PVV typically occur at the confluence of the veins adjacent to the left atrium.
They are sometimes classified into three morphological types: saccular, tortuous and confluent.

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary vein morphology [HP:0030968]

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Hereditary haemorrhagic telangiectasia

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added as HP_0034389
